components/autofill_strings.grdp

```diff
@@ -279,7 +279,7 @@
   <if expr="not is_ios">
     <then>
       <message name="IDS_AUTOFILL_ENABLE_PAYMENTS_INTEGRATION_CHECKBOX_LABEL" desc="Label for the checkbox that controls the Autofill/Payments integration feature used for Autofill Home. 'Brave Pay' should not be translated as it is the product name. Sentence-Cased.">
-        Payment methods and addresses using Brave Pay
```

Comment:
What is this? We don't need to change the naming, we only need to hide the toggle completely.

Reply:
Thanks, I agree, we don't need this change anymore.
Reverted at f8ee5b6af8aac4a5da5cf59faa5ff801780f4b7a